test/lib/code_corps_web/controllers/conversation_controller_test.exs

```diff
@@ -65,4 +65,37 @@ defmodule CodeCorpsWeb.ConversationControllerTest do
       assert conn |> request_show(conversation) |> json_response(403)
     end
   end
+
+  describe "update" do
+    @tag authenticated: :admin
+    test "updates and renders chosen resource when data is valid", %{conn: conn, current_user: current_user} do
+      message = insert(:message)
+      conversation = insert(:conversation, user: current_user, message: message)
```

Comment:
This test will need modified using logic similar to the `index`'s `conversation_on_user_administered_project` when the policy is changed, rather than relying on `admin` here.